src/teleop_twist_joy.cpp

```diff
@@ -68,63 +68,73 @@ struct TeleopTwistJoy::Impl
 
 /**
  * Constructs TeleopTwistJoy.
- * \param nh NodeHandle to use for setting up the publisher and subscriber.
- * \param nh_param NodeHandle to use for searching for configuration parameters.
  */
-TeleopTwistJoy::TeleopTwistJoy(rclcpp::Node::SharedPtr & node)
+TeleopTwistJoy::TeleopTwistJoy() : Node("teleop_twist_joy_node")
 {
-  pimpl_ = new Impl();
+  pimpl_ = new Impl;
 
-  pimpl_->node = node;
-
-  pimpl_->cmd_vel_pub = node->create_publisher<geometry_msgs::msg::Twist>("cmd_vel",
+  pimpl_->cmd_vel_pub = this->create_publisher<geometry_msgs::msg::Twist>("cmd_vel",
     rmw_qos_profile_sensor_data);
-  pimpl_->joy_sub = node->create_subscription<sensor_msgs::msg::Joy>("joy",
+  pimpl_->joy_sub = this->create_subscription<sensor_msgs::msg::Joy>("joy",
     std::bind(&TeleopTwistJoy::Impl::joyCallback, this->pimpl_, std::placeholders::_1),
     rmw_qos_profile_sensor_data);
 
-  pimpl_->enable_button = 5;
-  node->get_parameter("enable_button", pimpl_->enable_button);
-
-  pimpl_->enable_turbo_button = -1;
-  node->get_parameter("enable_turbo_button", pimpl_->enable_turbo_button);
-
-  // TODO(clalancette): node->get_parameter(s) doesn't seem to
-  // support getting a map of values yet.  Revisit this once it does.
-  // if (nh_param->getParam("axis_linear", pimpl_->axis_linear_map))
-  // {
-  //   nh_param->getParam("axis_linear", pimpl_->axis_linear_map);
-  //   nh_param->getParam("scale_linear", pimpl_->scale_linear_map);
-  //   nh_param->getParam("scale_linear_turbo", pimpl_->scale_linear_turbo_map);
-  // }
-  // else
-  {
-    pimpl_->axis_linear_map["x"] = 5;
-    node->get_parameter("axis_linear", pimpl_->axis_linear_map["x"]);
-    pimpl_->scale_linear_map["x"] = 0.5;
-    node->get_parameter("scale_linear", pimpl_->scale_linear_map["x"]);
-    pimpl_->scale_linear_turbo_map["x"] = 1.0;
-    node->get_parameter("scale_linear_turbo", pimpl_->scale_linear_turbo_map["x"]);
+  // TODO(clalancette): node->get_parameter(s) doesn't support getting a map of
```

Comment:
Can you provide the link of the ticket requesting the feature in the comment here?

Reply:
Good idea, I just opened an issue and linked to it.